src/AppBundle/Resources/views/Ndr/DeputyExpense/_list.html.twig

```diff
@@ -1,30 +1,32 @@
 {% set translationDomain = "ndr-deputy-expenses" %}
 {% trans_default_domain translationDomain %}
 
-<table class="check-your-answers columns-2 push-double--bottom">
-    <thead>
-        <tr>
-            <th>{{ 'question' | trans({}, 'common') }}</th>
-            <th>{{ 'answer' | trans({}, 'common') }}</th>
-            <th>
-                <span class="visually-hidden">{{ 'actions' | trans({}, 'common') }}</span>
-            </th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr class="behat-region-paid-for-anything">
-            <td>
-                {{ 'existPage.form.paidForAnything.label' | trans(transOptions)  }}
-            </td>
-            <td>
-                {{ ndr.paidForAnything | capitalize }}
-            </td>
-            <td class="change-answer">
-                <a href="{{ path('ndr_deputy_expenses_exist', { 'ndrId': ndr.id, 'from': 'summary' }) }}" class="behat-link-edit">{{ 'edit' | trans({}, 'common') }}</a>
-            </td>
-        </tr>
-    </tbody>
-</table>
+<dl class="govuk-summary-list columns-2 push-double--bottom">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Question
```

Comment:
This should be `{{ 'question' | trans({}, 'common') }}`. It's something we're trying to get better at. There are a few of these, might be worth searching for.